### PR TITLE
Fix identity function code example

### PR DESCRIPTION
**Pull request by danielstocks**
Hello,

I found a typo in the tutorial in regards to identity functions. Didn't raise an issue here as per contributions guidelines, but mentioned it to @sabine on [Bluesky](https://bsky.app/profile/roflwtfbbq.bsky.social/post/3lbwozdogss2i)

Anyway, I've suggested an edit in this PR to make more sense out of it.

Thanks in advance!

### CODE REVIEW

**Review by sabine — APPROVED** [review]
Makes a lot more sense now, thanks @danielstocks!

File: data/tutorials/language/0it_01_basic_datatypes.md [hunk 1]
```diff
@@ -341,14 +341,17 @@ The first expression is another anonymous function. It is the _identity_ functio
 
 Remember, the `'a` is a _type parameter_, so values of any type can be passed to the function and their type replaces the type parameter. The identity function has the same input and output type, whatever it may be.
 
-The two following expressions show that the identity function can apply to arguments of different types:
+The following example shows that the identity function can apply to arguments of different types:
 
 ```ocaml
-# let f = fun x -> x * x;;
+# let f = fun x -> x;;
 val f : int -> int = <fun>
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_01_basic_datatypes.md:348]:
```suggestion
val f : 'a -> 'a = <fun>
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Thanks, this needs to be fixed indeed. You also need to update f's type

File: data/tutorials/language/0it_01_basic_datatypes.md (hunk 1 above)
Comment (sabine) [reply data/tutorials/language/0it_01_basic_datatypes.md:348]: doh sorry I was too quick to merge 😅

Comment (sabine) [reply data/tutorials/language/0it_01_basic_datatypes.md:348]: going to fix it up tomorrow morning

**Pull request merged**